src/components/PopupWithForm.js

```diff
@@ -18,12 +18,18 @@ export default class PopupWithForm extends Popup {
     this._inputEls.forEach((input) => {
       this._inputValues[input.name] = input.value;
     });
+
     return this._inputValues;
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`